### ISSUE TITLE
Delay loading between each image

### ISSUE THREAD

**Issue by skillmatic-co** [tuupola/jquery_lazyload#107]
More of a question of possibility or a feature request, but it would be nice if I could add a variable that is an amount of time to wait in between loading the next image in the DOM. 

When I scroll and say 3 images appear above the page fold, I want a fading in from left to right to happen. Image on the left fades in first, waits the specified time delay, loads middle image, waits the specified time delay, and loads the third image.

Basically just adding a "wait" action in between each image load.

This would give a nice "flowing" effect to the loading in of images, almost so  your eyes go to the first one loaded first.

If this already exists or is possible, please let me know.
